BasementRenovator.py

```diff
@@ -1874,8 +1885,15 @@ def remove(self):
         self.scene().removeItem(self)
 
     def mouseReleaseEvent(self, event):
-        self.hideWeightPopup()
-        QGraphicsItem.mouseReleaseEvent(self, event)
+        e = self.entity
+        if (
+            event.button() == Qt.MiddleButton
+            and e.Type in Entity.ENTITIES_WITH_PROPERTIES
+        ):
+            EntityMenu(e)
+        else:
+            self.hideWeightPopup()
+            QGraphicsItem.mouseReleaseEvent(self, event)
```

Comment:
I think we want to process this code regardless, not sure if that causes issues
```suggestion
        self.hideWeightPopup()
        QGraphicsItem.mouseReleaseEvent(self, event)
```